### ISSUE TITLE
Styling of the active file via RStudio Addin removes EOF line break

### ISSUE THREAD

**Issue by lorenzwalthert** [r-lib/styler#310]
Styling the active file should make sure that there is always exactly one blank line at the end of the file.
Reference: https://github.com/r-lib/styler/issues/309#issuecomment-354665292.

**Comment by lorenzwalthert** [comment]
This fix provided in #311 was obviously not thought through. We should only append the empty string when there is no empty string already. Otherwise 
```r
identical(style_via_addin(x), style_via_addin(style_via_addin(x)))
#> FALSE
```

**Comment by lorenzwalthert** [comment]
Closed with #435.
